lib/offers_create/create_offer_view.dart

```diff
@@ -34,44 +35,88 @@ class CreateOffersFormState extends State<CreateOffersForm> {
   int selectedClassNo = 1;
   SchoolName selectedSchoolname = SchoolName.Schule1;
+  String selectedFirstSchoolday = DateTime.now().toString();
+  String selectedSchoolId = "1"; //hart gecoded
+  final dynamic defaultValue = 'default';
+
+  TextEditingController _dateController = TextEditingController();
 
   @override
   Widget build(BuildContext context) => Column(
         children: [
           Row(
-            children: <Widget>[
+            children: [
               Padding(
                 padding: const EdgeInsets.all(30),
                 child: DropdownButton<SchoolType>(
-                  items: SchoolType.values.map((type) => DropdownMenuItem(value: type, child: Text(type.name))).toList(),
-                  onChanged: (newItem) => setState(() {
-                    selectedSchooltype = newItem!;
-                  }),
+                  items: SchoolType.values
+                      .map((type) =>
+                          DropdownMenuItem(value: type, child: Text(type.name)))
+                      .toList(),
+                  onChanged: (newItem) {
+                    setState(() {
+                      selectedSchooltype = newItem ?? defaultValue;
+                    });
+                  },
                   value: selectedSchooltype,
                 ),
               ),
               Padding(
                 padding: const EdgeInsets.all(30),
                 child: DropdownButton<SchoolName>(
-                  items: SchoolName.values.map((type) => DropdownMenuItem(value: type, child: Text(type.name))).toList(),
-                  onChanged: (newName) => setState(() {
-                    selectedSchoolname = newName!;
-                  }),
+                  items: SchoolName.values
+                      .map((type) =>
+                          DropdownMenuItem(value: type, child: Text(type.name)))
+                      .toList(),
+                  onChanged: (newName) {
+                    setState(() {
+                      selectedSchoolname = newName ?? defaultValue;
+                    });
+                  },
                   value: selectedSchoolname,
                 ),
               ),
               Padding(
                 padding: const EdgeInsets.all(30),
                 child: DropdownButton<int>(
-                  items: List<int>.generate(13, (i) => i + 1).map((no) => DropdownMenuItem(value: no, child: Text(no.toString()))).toList(),
-                  onChanged: (newNo) => setState(() {
-                    selectedClassNo = newNo!;
-                  }),
-                  value: selectedClassNo,
+                  items: List<int>.generate(13, (i) => i + 1)
+                      .map((no) => DropdownMenuItem(
+                          value: no, child: Text(no.toString())))
+                      .toList(),
+                  onChanged: (newNo) {
+                    setState(() {
+                      selectedClassNo = newNo ?? defaultValue;
```

Comment:
in zeile 36 hast du ja schon eine default-klassenstufe eingeführt. bitte verwende die wieder